wca-regulations.md

```diff
@@ -145,6 +145,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3l1) The logo must be placed on a center piece. Exceptions for puzzles that do not have center pieces:
         - 3l1a) For Pyraminx and 2x2x2, the logo may be on any piece.
         - 3l1b) For Square-1, the logo must be on a piece in the equatorial slice.
+        - 3l1c) For Clock, the logo must be anywhere on the puzzle, as long as the clock faces remain uncovered.
```

Comment:
`Must` $\to$ `may`.

Mybe also worth adding `inner` to make it match the use from [10h1](https://www.worldcubeassociation.org/regulations/#10h1)?